DNN Platform/HttpModules/Membership/MembershipModule.cs

```diff
@@ -268,5 +249,25 @@ private void OnPreSendRequestHeaders(object sender, EventArgs e)
                 }
             }
         }
+
+        private void Context_PreRequestHandlerExecute(object sender, EventArgs e)
```

Comment:
I assume this was moved here due to the new sequence requirements for the localization process?  Might we give this method a bit better name for the purpose, although the name is correct as to what event it responds to, the WHY this is important isn't here and with no comments I'm concerned that a few years down the road we will all forget again

Reply:
The problem I'm trying to solve is that membership (AuthenticateRequest) must run before this. PreRequestHandlerExecute is the last event before main processing begins, so I moved this code to run at that time so we know who the logged in user is at this point.

Reply:
https://docs.microsoft.com/en-us/troubleshoot/aspnet/http-modules-handlers

Reply:
BTW, I think the naming is not all that bad given that it mimics what we do for event handlers. The original name had the same convention.

Reply:
I guess my point here is, yes the method name is “consistent”. But this was found after hours of debugging and there is nothing here that says WHY we do this. 

Renaming the method, or adding comments to save future self from pain was my point here.

Reply:
It's a good idea to comment. I'll add some.